agents/twenty_questions/agent.yaml

```diff
@@ -4,6 +4,9 @@ description: |
   The `twenty_questions` agent plays the 20 Questions game against you!
 
   tags: simple, no-fewshot, fun
+tests:
+  - query: Hi there, can we play?
+    expected: The response should be similar to "Yes, let's play! Pick a category and I'll think of something within it."
```

Comment:
Does 20 Questions use categories?

Reply:
Well, this is the response from the agent if you ask it "Hi there, can we play".

Reply:
Maybe this is incorrect behavior. I think this agent needs a little work, tbh....